test/v3/USTBv3.t.sol

```diff
@@ -81,13 +97,164 @@ contract USTBv3Test is SuperstateTokenTestBase {
             ITransparentUpgradeableProxy(address(tokenProxy)), address(tokenImplementation), ""
         );
 
-        token = USTB(address(tokenProxy));
         // No initialization needed for V3
+        token = USTB(address(tokenProxy));
+        tokenV3 = USTB(address(token));
+
+        // Set up oracle
+        oracle = new SuperstateOracle(address(this), address(tokenV3));
+        oracle.addCheckpoint(1726779600, 1726779601, 10_374_862, false);
+
+        vm.warp(1726866001);
+
+        oracle.addCheckpoint(uint64(1726866000), 1726866001, 10_379_322, false);
+
+        // Configure token with oracle
+        tokenV3.setOracle(address(oracle));
```

Comment:
do we want to have an `initializeV3()` method that does this in one txn?

Reply:
no because some tokens wont set an oracle because the feature will be disabled

Reply:
we may however want initializev3 for the stuff you're doing, but ill let you handle that